src/development/LocalTest/Controllers/HomeController.cs

```diff
@@ -247,6 +250,12 @@ private List<SelectListItem> GetAuthenticationLevels(int defaultAuthLevel)
         {
             return new()
             {
+                new()
+                {
+                    Value = "-1",
+                    Text = "Ikke autentisert",
+                    Selected = defaultAuthLevel == -1
```

Comment:
Oh, so this is why you need an `int`. I was thinking that `Value` could be more verbose, but this is fine.

Reply:
Yeah, and following the pattern made sense:)